packages/opam-publish/opam-publish.2.1.0/opam

```diff
@@ -14,13 +14,13 @@ depends: [
   "cmdliner"
   "dune" {>= "1.0"}
   "lwt_ssl"
+  "tls"
   "ocaml" {>= "4.03.0"}
   "opam-core" {>= "2.1.0~rc"}
   "opam-format" {>= "2.1.0~rc"}
   "opam-state" {>= "2.1.0~rc"}
-  "github" {>= "2.0.0" & < "4.3.0" | >= "4.3.2"}
-  "github-unix" {>= "2.0.0" & < "4.3.0" | >= "4.3.2"}
-  ("ssl" {= "0.5.5"} | "tls")
```

Comment:
This is weird: lwt_ssl requires ssl in any case, so this was forcing that specific ssl version to be installed. What is the use of tls then, isn't that redundant when you have to install ssl anyway?

Reply:
mmh, interesting. No idea. I'll let @AltGr reply

Reply:
`ssl | tls` was there before the dep to `lwt_ssl` was added so it's probably just a remain of something old ?
See 3d01aa43ac11902e6e2149cea5e8a7a4072dd290 (@rjbou do you remember anything about the ssl version constraint ?)

Reply:
No, I just added constraints, it originates from https://github.com/ocaml-opam/opam-publish/commit/1bad73a14fd68dc4fbee75f5b0cd1ef9ad4af53d

Reply:
yes, indeed at that point we didn't rely on lwt, and either was required to have the `github` package compiled with the required features IIRC. I guess the `ssl | tls` can be dropped now. @rjbou by "no" you meant you do not remember the reason for the version constraint ?

Reply:
It was because an error in `ssl.0.5.6`, it is fixed in [`ssl.0.5.7`](https://github.com/savonet/ocaml-ssl/pull/41), so we can change the constraint.
* [opam-publish PR](https://github.com/ocaml-opam/opam-publish/pull/71)
* [opam-publish issue](https://github.com/ocaml-opam/opam-publish/issues/74)
* [ssl issue](https://github.com/savonet/ocaml-ssl/issues/40)

Reply:
Thanks! I have added a conflict